app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -316,20 +316,19 @@ protected function getMultilineFieldConfig($attributeCode, array $attributeConfi
     protected function getDefaultValue($attributeCode)
     {
         switch ($attributeCode) {
```

Comment:
Wouldn't it be better if you return `null` straight away, at the beginning of the method in case where there is no customer? I guess it should reduce complexity that way, since you would not have to check for customer every time.